contracts/SimpleMultiSignature.sol

```diff
@@ -33,7 +33,12 @@ contract SimpleMultiSignature {
 
   function changeOwner(address newOwner, address lastOwner) internal returns (bool) {}
 
-  function changeThreshold(uint256 newThreshold) internal returns (bool) {}
+    function changeThreshold(uint256 newThreshold) internal returns (bool) {
```

Comment:
change argument type for uint16 since threshold is uint16 and add the modifier we created